opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -192,6 +194,14 @@ configure() {
       export "$assignment"
     done < "$MINION_PROCESS_ENV_CFG"
   fi
+  if [[ -f "$MINION_SERVER_CERTS_CFG" ]]; then
+    cp "$JAVA_HOME/lib/security/cacerts" "$CACERTS"
+    export JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStore=$CACERTS -Djavax.net.ssl.trustStorePassword=changeit"
```

Comment:
Should we be using something besides `changeit`?

Reply:
The added server certificates are public, i.e. they do not contains secrets. In this regard, security is not changed by this pull request. Securing the trust store may be worth another ticket.